Gopkg.lock

```diff
@@ -188,6 +188,15 @@
   revision = "2dd44038f0b95ae693b266c5f87593b5d2fdd78d"
   version = "v2.5.0"
 
+[[projects]]
```

Comment:
We should avoid SHA-based - can you create a branch if master is not stable ?

Reply:
For what? The go control plane is effectively an api. And we need to track which api sha we pick as we won’t be enabling or supporting all features. Having api sha helps us nail down the Envoy sha that supports it.

Reply:
Right, "v2" urgently needs to be stabilized yet, but it's not fully baked, so let's bear with the churn by using SHAs.

Reply:
So this sha matches the sha of the envoy importend by the proxy - which we also import by a different sha.  Does whoever updates proxy envoy sha knows to update this sha as well ? Or what is the process to avoid SHA out of date with the proxy build ?

We have had too many problems with out-of-date SHAs - if the API is changing randomly 
and breaks we should not attempt to use it, and I would rather have the build break and 
know about uncompatible API changes, so we can deal with proxy upgrade issues and
possible breakages on miss-matched versions ( since we can't upgrade proxy and 
pilot at the same time )

Reply:
Envoy master is changing rapidly, and yes, there'll be times when go-control-plane and proxy are incompatible. I can start co-versioning the library with envoy releases, once there's some meat there. Envoy data-plane-api has the exact same issue, and it was not a problem up to now.